boards/arm/ubx_bmd345eval_nrf52840/board.c

```diff
@@ -11,7 +12,7 @@
 #define MODE_PIN	36	/* P1.04 */
 #define A_SEL_PIN	34	/* P1.02 */
```

Comment:
These values are used as parameters in calls to `gpio_pin_configure()` so they should be 4 and 2, respectively. If it works in the current form, it only works by chance. This assertion: https://github.com/zephyrproject-rtos/zephyr/blob/0500d75c3a9ad228867529a650147534dc027e1c/include/drivers/gpio.h#L689-L690 should actually fail for both those calls.

Reply:
I seem to be missing a CMakeLists.txt file, so board.c never got pulled into my tests, and the default state is low.
Working through this now. This is not returning a valid device:
```        mode_asel_port_dev = device_get_binding(DT_LABEL(DT_NODELABEL(gpio1)));```

Reply:
Converted this to draft mode until this is sorted.